README.md

```diff
@@ -356,7 +356,8 @@ chmod +x run_script.sh
 ## Run Pylint Check Locally
 This project uses [Pylint](https://pypi.org/project/pylint/) tool for static code analysis.
 Pylint analyses your code without actually running it.
-It checks for errors, enforces, coding standards, looks for code smells etc. 
+It checks for errors, enforces, coding standards, looks for code smells etc.
+We do exclude the `tests/` file from the pylint check.
```

Comment:
When I run the command
`pylint $(git ls-files '*.py')`
I got a list of violations from files on **tests** path.

Reply:
I see. Nice catch! I suggest in this commit:
to change our command into `pylint $(git ls-files '*.py' | grep -v '^tests/')`. That will cause, that all the workflows will work correctly. And we still can use pylint whenever we want on our test files. Same pattern I want to use with black tool.

Reply:
I do not like this solution.

Reply:
I tested this for pylint:

[MASTER]
ignore-paths=tests

Reply:
for black use
force-exclude = '''test'''

Reply:
Alright, fixed.